fluster/platformdirs.py

```diff
@@ -0,0 +1,64 @@
+# Fluster - testing framework for decoders conformance
+# Copyright (C) 2023, Fluendo, S.A.
+#  Author: Ruben Gonzalez <rgonzalez@fluendo.com>, Fluendo, S.A.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public License
+# as published by the Free Software Foundation, either version 3
+# of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library. If not, see <https://www.gnu.org/licenses/>.
+
+# With the goal of avoid deps. This is a mini version of needed part of
+# the https://github.com/platformdirs/platformdirs lib
+
+import os
+import sys
+from typing import Callable
+
+try:
+    import platformdirs
```

Comment:
There are no benefits of using `platformdirs` if we already provide a working implementation. This can also have side effects such as producing different results if `platformdirs` is installed or not.

Reply:
The `platformdirs` library for Windows is very complex with a lot of cornel cases like reading paths from the winreg.

What about only import `platformdirs` if Windows?

Reply:
On Windows you can reliably retrieve the data dir for resources with the `APPDATA` environment variable.

Reply:
> On Windows, we should only consider for now installation from PyPI.

Why? We are using it inside fluendo CI.

IMHO It is a very big requirement

Reply:
I prefer to drop the fallback to the `USE_SYSTEM_PLATFORMDIRS`. Done in a418f2a.

Reply:
> > On Windows, we should only consider for now installation from PyPI.
> 
> Why? We are using it inside fluendo CI.
> 
> IMHO It is a very big requirement

That is an  "uninstalled" use. We will not provide .exe installers for fluster on Windows, users will either use it uninstalled or installed through PyPI.